dynamometer-workload/src/main/java/com/linkedin/dynamometer/workloadgenerator/audit/AuditReplayThread.java

```diff
@@ -254,7 +258,16 @@ public FileSystem run() {
           fs.concat(new Path(src), dsts.toArray(new Path[] {}));
           break;
       }
+
+      String key = command.getSimpleUgi() + "_" + replayCommand.getType().toString();
       long latency = System.currentTimeMillis() - startTime;
+
+      try {
+        mapperContext.write(new Text(key), new LongWritable(latency));
```

Comment:
It's been a while since I've written an MR job, but I believe it's more efficient from a GC perspective to instantiate single instances of `Text` and `LongWritable` and then use their `set` methods to change their contents.